latest/examples/transformations/affine2d3d.json

```diff
@@ -0,0 +1,14 @@
+{
+    "coordinateSystems": [
+        { "name": "ij", "axes": [{"name": "i"}, {"name": "j"}] },
+        { "name": "xyz", "axes": [{"name": "x"}, {"name": "y"}, {"name": "z"}] }
+    ],
+    "coordinateTransformations": [ 
+        {
+            "type": "affine",
+            "affine": [1, 2, 3, 4, 5, 6, 7, 8, 9],
+            "input": "ij",
+            "output": "xyz"
+        }
+    ]
+}
```

Comment:
This is the first example where I really don't know at first glance what's going on...?

Reply:
Related: https://github.com/ome/ngff/issues/146#issuecomment-1270117494 (please @bogovicj close the issue if the nested representation is adopted)